handler/src/main/java/io/netty/handler/ssl/JdkSslClientContext.java

```diff
@@ -256,22 +258,22 @@ public JdkSslClientContext(File trustCertChainFile, TrustManagerFactory trustMan
         }
     }
 
-    JdkSslClientContext(X509Certificate[] trustCertChain, TrustManagerFactory trustManagerFactory,
+    JdkSslClientContext(X509Certificate[] trustCertCollection, TrustManagerFactory trustManagerFactory,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Constructor has 11 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS00107)